packages/graphql-cli-validate-magento-pwa-queries/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "graphql-cli-validate-magento-pwa-queries",
   "version": "1.0.0",
+  "publishConfig": {
+    "access": "public"
```

Comment:
Did we want to release this publicly? I feel like @zetlen may have said otherwise at some point but I don't recall exactly.

Reply:
I think he meant not be in scope @magento org but will wait for @zetlen to confirm.